
Sun Jul 01 17:42:25 2018: Request 90895 was acted upon.
 Transaction: Queue changed from 0-Help to 0-SDSC by buskuehl
       Queue: 0-SDSC
     Subject: XUP: phono3py
       Owner: buskuehl
  Requestors: bdebn001@ucr.edu
      Status: open
 Ticket <URL: https://tickets.xsede.org/Ticket/Display.html?id=90895 >


[Ticket created from XUP by bdebnath]
[From: BISHWAJIT DEBNATH]
[System: Comet]
[Category: Software]
Hi, 
I am trying to install phono3py (https://atztogo.github.io/phono3py/install.html#installation-using-pip).

I have used my own python-3 environment (source activate py36) and conda:
conda install -c atztogo phono3py

It installed without any error. But when I run it, it issues following error:
 /lib64/libc.so.6: version `GLIBC_2.14' not found 
I have attached the screenshot.

Can you please help me in identifying and fixing the issue? 

Thanks in advance.

Bishwajit 
